modules/standard/Path.chpl

```diff
@@ -43,6 +42,7 @@
    :proc:`commonPath`
    :proc:`realPath`
    :proc:`file.realPath`
+   :proc:`normPath`
```

Comment:
Missed this the first time, only saw it when reading the generated documentation.  This should be organized similarly to the rest of the file (i.e. normPath should come before realPath)